dspace-server-webapp/src/main/java/org/dspace/app/rest/ItemAddBundleController.java

```diff
@@ -183,6 +187,7 @@ public ItemRest updateLicenseForBundle(@PathVariable UUID uuid,
         }
 
         itemService.update(context, item);
+        provenanceService.editLicense(context, item, !Objects.isNull(clarinLicense));
```

Comment:
edit or update?

Reply:
Agree, update is more consistent. Changed